transaction.js

```diff
@@ -10,6 +10,7 @@ var MessageStream = require('./messagestream');
 var trans = exports;
 
 var MAX_HEADER_LINES = config.get('max_header_lines') || 1000;
+var REMOVE_CARRIAGE_RETURN = config.get('remove_carriage_return') || 'true';
```

Comment:
What happens here if the file `remove_carriage_return` contains `false`?

Some test coverage of this might be a good idea.